index.js

```diff
@@ -277,6 +277,81 @@ var MysqlDriver = Base.extend({
     }).nodeify(callback);
   },
 
+  createTable: function(tableName, options, callback) {
+    log.verbose('creating table:', tableName);
+    var columnSpecs = options;
+    var tableOptions = {};
+
+    if (options.columns !== undefined) {
+      columnSpecs = options.columns;
+      delete options.columns;
+      tableOptions = options;
+    }
+
+    var ifNotExistsSql = "";
+    if(tableOptions.ifNotExists) {
+      ifNotExistsSql = "IF NOT EXISTS";
+    }
+
+    var engineSql = '';
```

Comment:
For an example see the cockroachdb driver I just lately added: https://github.com/db-migrate/cockroachdb/blob/master/index.js